### PR TITLE
Removed SendUsageTask + Minecraft Version Update

### PR DESCRIPTION
**Pull request by erkam2002**
This removes only SendUsage to stats.pocketmine.net

### CODE REVIEW

**Comment by erkam2002** [comment]
Added new Commit

File: src/pocketmine/network/protocol/Info.php [hunk 1]
```diff
@@ -20,20 +20,22 @@
...
 interface Info {
 	/**
 	 * Actual Minecraft: PE protocol version
 	 */
 
 	const CURRENT_PROTOCOL = 113;
 	const ACCEPTED_PROTOCOLS = [110, 111, 112, 113];
-	const MINECRAFT_VERSION = ["v1.1.0", "v1.1.1", "v1.1.2", "v1.1.3"];
-	const MINECRAFT_VERSION_NETWORK = "1.1.3";
+	const MINECRAFT_VERSION = ["v1.1.0", "v1.1.1", "v1.1.2", "v1.1.3", "1.1.4"];
```
Comment (Leo3418) [inline src/pocketmine/network/protocol/Info.php]: Please make it consistent with other version tags.

Comment (erkam2002) [inline src/pocketmine/network/protocol/Info.php]: ? My english is bad

Comment (erkam2002) [inline src/pocketmine/network/protocol/Info.php]: I`m not pro on Github idk how i can do this in 1 fork if i create a pr it compares automatically

**Comment by akaSnapy** [comment]
Crap, 1.1.4 uses protocol 113.

**Comment by akaSnapy** [comment]
This pr is useless as fuck, don't remove stats tracking wtf

**Comment by erkam2002** [comment]
I dont changed protocol e.t.c pocketmine stats are not used and will not used more from PocketMine

**Comment by DerddyDert** [comment]
@erkam2002 You missed the "v"

```
+	const MINECRAFT_VERSION = ["v1.1.0", "v1.1.1", "v1.1.2", "v1.1.3", "1.1.4"];
Should read
+	const MINECRAFT_VERSION = ["v1.1.0", "v1.1.1", "v1.1.2", "v1.1.3", "v1.1.4"];
```

File: src/pocketmine/network/protocol/Info.php (hunk 1 above)
Comment (Leo3418) [inline src/pocketmine/network/protocol/Info.php]: No, you should learn how to use branches, not only forks. It's fundamental knowledge of Git.

**Comment by erkam2002** [comment]
But now..

**Comment by DerddyDert** [comment]
Now what?

**Comment by erkam2002** [comment]
Nothing i mean the v

**Comment by erkam2002** [comment]
Will this merged or not?

**Comment by BajanVlogs** [comment]
NO DONT USE THIS U WILL BREAK GENISYSPRO

**Comment by erkam2002** [comment]
Why will this break?

**Comment by SuperXingKong** [comment]
Why? Thereis no point in doing this...

**Pull request closed without merging**